### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by eyuell**

### CODE REVIEW

**Review by MarcusAronsson1 — CHANGES_REQUESTED** [review]
* Please correct the spelling of the Developers names, I see at least 1 wrong one.

* At line 28, "The use of Arduino and Android libraries would essential", is this a correct sentence? Also, the sentence after starts with "And", which is not a good thing.

**Review by eyuell — COMMENTED** [review]
reviews accepted

**Review by AdamKlevfors — DISMISSED** [review]
Update approved

File: README.md [hunk 1]
```diff
@@ -17,6 +25,22 @@ Each year, over [5000](http://www.the-monitor.org/en-gb/reports/2017/landmine-mo
...
+* Jacob Olsson
+* Ilja Pavlov
+
+### Developers
+* Adam Klevfors
+* August Axelsson
+* Eyuell Hailemichael
+* Hanna Maltkvist
+* Marcus Aronsson
+* Oscar Lundberg
```
Comment (MarcusAronsson1) [inline README.md]: Wrong spelling here

File: README.md [hunk 2]
```diff
@@ -17,6 +25,22 @@ Each year, over [5000](http://www.the-monitor.org/en-gb/reports/2017/landmine-mo
...
+### Client Team
+* Dimitrios Platis
+* Jacob Olsson
+* Ilja Pavlov
+
+### Developers
+* Adam Klevfors
+* August Axelsson
+* Eyuell Hailemichael
+* Hanna Maltkvist
```
Comment (MarcusAronsson1) [inline README.md]: Wrong spelling here

File: README.md [hunk 3]
```diff
@@ -17,6 +25,22 @@ Each year, over [5000](http://www.the-monitor.org/en-gb/reports/2017/landmine-mo
 * Odometers
 
 ### Software
-* [smartcar](https://github.com/platisd/smartcar_shield) - Library for arduino based vehicles
+The use of Arduino and Android libraries would be essential. And the developers are aiming to develop necessary methods to complete the project.
```
Comment (MarcusAronsson1) [inline README.md]: Don't start sentences with "And".

**Review by MarcusAronsson1 — CHANGES_REQUESTED** [review]
Still more changes to be made.

**Review by MarcusAronsson1 — APPROVED** [review]
Nice!

**Pull request merged**